### DISCUSSION TITLE
docker: invalid reference format

### DISCUSSION THREAD

**Discussion by jagujar** [lobehub/lobehub#4036] [General Question]
How do I fix the error?
Solution: docker run -d -p 3210:3210 -e OPENAI_API_KEY=sk-xxxx -e ACCESS_CODE=lobe66 --name lobe-chat lobehub/lobe-chat
You need to enter this command, before that you need to add docker to your environment variables.

**Comment by SpeedupMaster** [comment]
To run the Docker command you provided, you need to ensure that Docker is installed and properly configured on your system. Here are the steps to add Docker to your environment variables and run the command:

### Step 1: Install Docker
If Docker is not already installed on your system, you can install it by following the official Docker installation guide for your operating system:

- **Windows**: [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/)
- **macOS**: [Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/install/)
- **Linux**: [Docker Engine on Linux](https://docs.docker.com/engine/install/)

### Step 2: Verify Docker Installation
After installing Docker, open a terminal (or command prompt on Windows) and verify that Docker is installed correctly by running:

```bash
docker --version
```

This should display the installed Docker version.

### Step 3: Add Docker to Environment Variables (Optional)
On most systems, Docker is automatically added to the environment variables during installation. However, if you encounter issues where Docker commands are not recognized, you may need to manually add Docker to your PATH.

#### On Windows:
1. Open the Start Menu and search for "Environment Variables".
2. Click on "Edit the system environment variables".
3. In the System Properties window, click on the "Environment Variables" button.
4. In the "System variables" section, find the `Path` variable and click "Edit".
5. Add the path to the Docker executable directory (e.g., `C:\Program Files\Docker\Docker\resources\bin`).
6. Click "OK" to save the changes.

#### On macOS/Linux:
1. Open your terminal.
2. Edit your shell configuration file (e.g., `.bashrc`, `.zshrc`, etc.) using a text editor:

    ```bash
    nano ~/.bashrc
    ```

3. Add the following line to the file:

    ```bash
    export PATH="/usr/local/bin:$PATH"
    ```

4. Save the file and exit the editor.
5. Reload the shell configuration:

    ```bash
    source ~/.bashrc
    ```

### Step 4: Run the Docker Command
Now that Docker is installed and configured, you can run the provided Docker command:

```bash
docker run -d -p 3210:3210 -e OPENAI_API_KEY=sk-xxxx -e ACCESS_CODE=lobe66 --name lobe-chat lobehub/lobe-chat
```
